edc-tests/e2e-fixtures/src/testFixtures/java/org/eclipse/tractusx/edc/tests/participant/TractusxParticipantBase.java

```diff
@@ -183,6 +183,21 @@ public void retireProviderAgreement(String agreementId) {
                 .statusCode(204);
     }
 
+    public void retireProviderAgreementBadRequest(String agreementId) {
```

Comment:
please just have a single `retireProviderAgreement` method that returns either the state or the `ValidatableResponse` and let test explicitly add their expectations, as I pointed in the previous review

note: the name says "bad request" but the status code verified is 404

Reply:
misunderstood, sorry, changed in [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1631/commits/b54f8c3a7cd2e664633b06e778c1cb5d6bf32c09).
regarding the "note" you're right, should've been a notfound.